src/Executor/ExecutionResult.php

```diff
@@ -158,8 +158,14 @@ public function toArray(int $debug = DebugFlag::NONE): array
         $result = [];
 
         if ($this->errors !== []) {
+            /**
+             * @phpstan-var ErrorsHandler $defaultErrorsHandler
+             *
+             * @phpstan-ignore varTag.nativeType (phpstan cannot type callables yet)
+             */
+            $defaultErrorsHandler = static fn (array $errors, callable $formatter): array => \array_map($formatter, $errors);
             $errorsHandler = $this->errorsHandler
-                ?? static fn (array $errors, callable $formatter): array => \array_map($formatter, $errors);
+                ?? $defaultErrorsHandler;
```

Comment:
Can't we go back to inlining this and ignoring the error there? What you did has (slight) performance implications.

Reply:
I'd bet it gets inlined in opcode but sure

Reply:
Also, I did it in order to narrow the error to the default handler only because it was kinda confusing whether it complains about that or `$this->errorsHandler`. 

Reverting.

Reply:
I managed to fix it accidentally 😅